### DISCUSSION TITLE
Welcome to JAXNS Discussions!

### DISCUSSION THREAD

**Discussion by Joshuaalbert** [Joshuaalbert/jaxns#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with JAXNS.

**Comment by ekerrison** [comment]
Hello! I'm a bit of a Bayesian beginner. I'm a PhD student in radio astronomy and I'm just starting to do nested sampling using JAXNS to model the radio SEDs of various galaxies.
